assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -97,6 +98,11 @@ export default Component.extend({
     );
 
     this.appEvents.on("chat:cancel-message-selection", this, "cancelSelecting");
+
+    this.set(
+      "showCompressBtn",
```

Comment:
```suggestion
      "showCloseFullScreenBtn",
```

Minor suggestion because I feel like `compress` doesn't really describe what the button actually does on click.

Reply:
Ya I like that better too.